module/zfs/arc.c

```diff
@@ -3195,8 +3197,11 @@ arc_shrink(int64_t to_free)
 		ASSERT((int64_t)arc_p >= 0);
 	}
 
-	if (arc_size > arc_c)
+	if (arc_size > arc_c) {
```

Comment:
this can be simplified by putting the mutex_exit() above this conditional and removing the else

Note: for style reasons, it is best to not mix {} with non-{} in an if-then-else